### PR TITLE
Home page tile styling added

### PR DESCRIPTION
**Pull request by dandwal**
### What

- JS for show and hide show more and fewer content
- New font size (30) added

### How to review

1. Checkout instructions on here: https://github.com/ONSdigital/dp-frontend-homepage-controller/pull/7 
2. Then add a replace line to your go mod file pointing to the latest commit on this branch (or checkout this branch and point to your local copy of sixteens.
3. Checkout the following PR for the renderer https://github.com/ONSdigital/dp-frontend-renderer/pull/389 
4. Run zebedee reader 

### Who can review

Anyone except me but might be worth other frontenders giving this a review

### CODE REVIEW

File: scss/v2/components/tile.scss [hunk 1]
```diff
@@ -0,0 +1,160 @@
+.tile {
+  font-size: 18px;
+  line-height: 1.78;
+  color: $grey2;
+  overflow: hidden;
+  box-shadow: 0 2px 0 0 rgba($grey1, .3);
+  background-color: $grey5;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-bottom: 16px;
```
Comment (rav-pradhan) [inline scss/v2/components/tile.scss:10]: A stylistic thing more than anything, but seeing as we do it with a few `margin` properties below, perhaps we can format the `padding` here similarly?

File: scss/v2/components/tile.scss [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  }
+
+  &__button {
+    margin: 8px 0 8px 0;
+    padding: 8px 0 8px 0;
+    background-color: $ocean-blue;
+    font-size: 14px;
+    font-weight: bold;
+    text-align: center;
+    color: #ffffff;
```
Comment (rav-pradhan) [inline scss/v2/components/tile.scss:124]: Perhaps we can update this to use the variable name :)

File: scss/v2/components/tile.scss [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    line-height: 1.78;
+    color: $ocean-blue;
+    text-decoration: underline;
+    margin-left: 16px;
+
+    &:hover {
+      box-shadow: 0 4px 0 0 $black;
+      background-color: $sun-yellow;
+      color: $black;
+    }
```
[the remark this answers is not in the document]
Comment (dandwal) [reply scss/v2/components/tile.scss:96]: Will wait for clarification on this but push everything else for now

[the remark this answers is not in the document]
Comment (armstrongb) [reply scss/v2/components/tile.scss:102]: Links in the stat blocks should behave the same as elsewhere on the page for  `:focus` and `:hover` states. Thanks for checking.

**Pull request merged**